lessons/fr/basics/strings.md

```diff
@@ -0,0 +1,188 @@
+---
+version: 1.2.0
+title: Chaines
+---
```

Comment:
We use a different syntax for this header information now. I think this should do the trick and build correctly:

```suggestion
%{
  version: "1.2.0",
  title: "Chaines",
  excerpt: """
  Strings, Char Lists, Graphemes and Codepoints.
  """
}
```